apps/web/vibes/soul/sections/maintenance/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { clsx } from 'clsx';
+import Link from 'next/link';
+
+interface Props {
+  className?: string;
+  logo?: React.ReactNode;
```

Comment:
I think this should be the same `logo` prop type that's used in the `Navigation` and `Footer` that uses the `Logo` component.

Reply:
@apledger Fair enough!